pages/[id]/edit.js

```diff
@@ -0,0 +1,177 @@
+import styled from "styled-components";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import Map from "../../components/map/index";
+import Modal from "../../components/modal/index";
+import { useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import apiUrl from "next-api-url";
+
+export default function EditCard({
```

Comment:
Same here: Why does this have to be global state that comes from the _app.js?
It could probably be eliminated as state completely or turned into local state.